desc/equilibrium/coords.py

```diff
@@ -453,43 +448,57 @@ def _partial_sum(lmbda, L_lmn, omega, W_lmn, iota):
     return lmbda_minus_iota_omega, jnp.fft.rfftfreq(grid.num_theta, 1 / grid.num_theta)
 
 
-@partial(jit, static_argnames=["tol", "maxiter"])
-def _map_clebsch_coordinates(
+@partial(jit, static_argnames=["inbasis", "outbasis", "tol", "maxiter"])
+def _map_poloidal_coordinates(
     iota,
-    alpha,
+    poloidal,
     zeta,
     L_lmn,
     lmbda,
+    varepsilon=None,
+    inbasis=("rho", "alpha", "zeta"),
+    outbasis=("rho", "theta", "zeta"),
```

Comment:
im a bit confused by inbasis and outbasis here. isn't the input radial coordinate always iota and input toroidal coordinate always zeta? and the output is always only the poloidal angle? I think it would be clearer if inbasis and outbasis were just single coordinates, not triplets (basically to label what `poloidal` is and what the output should be)

Reply:
I was trying to match the eq.map_coordinates api. i can change as you have requested to just one variable.